READ ME

```diff
@@ -0,0 +1,4 @@
+First we had a chat about what we liked and disliked on profile pages. We spoke about everything from color to color schemes
```

Comment:
Save read me files with with either '.txt' extension or more commonly '.md' extension for markdown.